# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/src/bindings.rs
# Commit: 548e4395ee1a
# Full Hash: 548e4395ee1a10700a729f93c0340d44ebe91ac7
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2019-09-13 15:46:31
# Regressor Bug: 1570081
# File Overlap Count: 1
# Description:
#   Bug 1570081 - Support changing the blob image visible rect in WebRender. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39911
# ==============================================================================

diff -r 8eeddedeedb8 -r 548e4395ee1a gfx/webrender_bindings/src/bindings.rs
--- a/gfx/webrender_bindings/src/bindings.rs	Thu Sep 12 21:27:06 2019 +0000
+++ b/gfx/webrender_bindings/src/bindings.rs	Thu Sep 12 15:54:50 2019 +0000
@@ -3236,9 +3236,8 @@
 extern "C" {
      // TODO: figure out the API for tiled blob images.
      pub fn wr_moz2d_render_cb(blob: ByteSlice,
-                               width: i32,
-                               height: i32,
                                format: ImageFormat,
+                               render_rect: &LayoutIntRect,
                                visible_rect: &DeviceIntRect,
                                tile_size: Option<&u16>,
                                tile_offset: Option<&TileOffset>,